kernel/mod_subst.ml

```diff
@@ -24,7 +24,7 @@ open Constr
    is the term into which we should inline. *)
 
 type delta_hint =
-  | Inline of int * constr option
+  | Inline of int * (Univ.AUContext.t * constr) option
```

Comment:
Why not use `in_univ_context`?

Reply:
That's not the same type, here we want an abstract context rather than a named one.